ManagerThing.md

```diff
@@ -0,0 +1,142 @@
+# WoT ManagerThing
+
+The purpose of this document is to explain ManagerThing that can deal remote Thing lifecycle management by co-working with ScriptingAPI.
+
```

Comment:
Perhaps some reformulation:
"This document presents a special Thing called `ManagerThing` that provides an interface to handle WoT scripts lifecycle management (install, update, run, stop, etc).